.github/CODEOWNERS

```diff
@@ -927,6 +927,12 @@
 # ServiceLabel: %Mgmt
 # AzureSdkOwners:                                                  @ArthurMa1978
 
+# PRLabel: %Carbon Optimization
+/sdk/carbon/Azure.ResourceManager.CarbonOptimization/                 @destinyboyer @liangchenmicrosoft
```

Comment:
`liangchenmicrosoft` also need write access to the repository.

Reply:
He has been accepted into the SG as of this morning. Once the pipelines that replicate permissions to from GitHub to CoreIdentity are unblocked I expect everything to be good.